packages/block-library/src/template-part/edit/placeholder/index.js

```diff
@@ -13,71 +13,91 @@ import { store as coreStore } from '@wordpress/core-data';
  * Internal dependencies
  */
 import TemplatePartSelection from '../selection';
+import PatternsSetup from './patterns-setup';
 
 export default function TemplatePartPlaceholder( {
 	area,
+	clientId,
 	setAttributes,
-	innerBlocks,
 } ) {
 	const { saveEntityRecord } = useDispatch( coreStore );
-	const onCreate = useCallback( async () => {
-		const title = __( 'Untitled Template Part' );
-		// If we have `area` set from block attributes, means an exposed
-		// block variation was inserted. So add this prop to the template
-		// part entity on creation. Afterwards remove `area` value from
-		// block attributes.
-		const record = {
-			title,
-			slug: 'template-part',
-			content: serialize( innerBlocks ),
-			// `area` is filterable on the server and defaults to `UNCATEGORIZED`
-			// if provided value is not allowed.
-			area,
-		};
-		const templatePart = await saveEntityRecord(
-			'postType',
-			'wp_template_part',
-			record
-		);
-		setAttributes( {
-			slug: templatePart.slug,
-			theme: templatePart.theme,
-			area: undefined,
-		} );
-	}, [ setAttributes, area ] );
+	const [ step, setStep ] = useState( 1 );
```

Comment:
I think we can use an `enum` like object for better readability here or plain strings.

Reply:
Sounds good!  Something along these lines? - 11c84b7

Reply:
Yes, but I think it's better to have the props lowercase and you can use `numbers` as values. ex: `initial: 1`